keps/sig-windows/3503-host-network-support-for-windows-pods/README.md

```diff
@@ -0,0 +1,882 @@
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+
+Documentation is available on [feature gate lifecycle] and expectations, as
+well as the [existing list] of feature gates.
+
+[feature gate lifecycle]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
+[existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
+-->
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: WindowsHostNetwork
+  - Components depending on the feature gate: kubelet
+- [ ] Other
+  - Describe the mechanism:
+  - Will enabling / disabling the feature require downtime of the control
+    plane?
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+###### Does enabling the feature change any default behavior?
+
+<!--
+Any change of default behavior may be surprising to users or break existing
+automations, so be extremely careful here.
+-->
+
+Enabling this feature gate will cause the kubelet to populate the CRI-API fields outlined above.
+It will be up to each container runtime to act on these fields.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+<!--
+Describe the consequences on existing workloads (e.g., if this is a runtime
+feature, can it break the existing applications?).
+
+Feature gates are typically disabled by setting the flag to `false` and
+restarting the component. No other changes should be necessary to disable the
+feature.
+
+NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
+-->
+
+If this feature is disabled after Windows Pod's that set `hostNetwork=true` have already been scheduled,
+if those Pod's get recreated they will not use the node's networking namespace.
+The impact of this will vary per workload.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+If this feature is reenabled then any Pod's that were created when the feature was disabled will be joined
+to the node's networking namespace if they get recreated.
+The impact of this will vary per workload.
+
+###### Are there any tests for feature enablement/disablement?
+
+No
```

Comment:
We should add tests for this, to make sure that enablement/disablement works. You can say something like "These will be added to kubelet to verify that the API is/is not populated based on the state of the feature gate."

Reply:
Can you change the text of the KEP to reflect this?

Reply:
whoops, I thought I did (but I forgot to push).
Updated now.